app/controllers/users_controller.rb

```diff
@@ -85,7 +85,7 @@ def confirm_destroy
   def full_delete
     set_tabs("account")
     if @user.email?
-      Users::SelfDeleteWorker.perform_async(@user.id)
+      Users::DeleteWorker.perform_async(@user.id)
```

Comment:
Renamed since both self deletion and moderator deletion defer to the service object, which this job wraps.